LibreNMS/OS.php

```diff
@@ -130,21 +130,22 @@ public function getCacheByIndex($oid, $mib = null, $snmpflags = '-OQUs')
      * DO NOT use numeric oids with this function! The snmp result must contain only one oid.
      *
      * @param string $oid textual oid
-     * @param string $mib mib for this oid
+     * @param string $mib mib for this oid (optional)
+     * @param string $depth depth for snmpwalk_group (optional)
      * @return array array indexed by the snmp index with the value as the data returned by snmp
      */
-    public function getCacheTable($oid, $mib = null)
+    public function getCacheTable($oid, $mib = null, $depth = null)
```

Comment:
$depth should default to 1, not null otherwise default is 0 instead of 1 like parent function.

Reply:
done